gbpservice/neutron/config/nsf.ini

```diff
@@ -0,0 +1,13 @@
+[DEFAULT]
```

Comment:
This file is not part of config_agent ,removed core and proxy from config_agent_modules branch